contracts/views/EulerGeneralView.sol

```diff
@@ -74,6 +74,12 @@ contract EulerGeneralView is Constants {
         uint eTokenBalanceUnderlying;
```

Comment:
yes, we want to start naming them LensV1, LensV2 etc. But here we actually need to discuss the migration, because current view will be broken with RiskManager upgrade.